packages/wallets/src/features/cfd/screens/ChangePassword/TradingPlatformChangePasswordScreens.tsx

```diff
@@ -88,12 +89,12 @@ const MT5ChangePasswordScreens: React.FC<MT5ChangePasswordScreensProps> = ({ pla
             <WalletsActionScreen
                 description={ChangePasswordScreens[activeScreen].bodyText}
                 descriptionSize='sm'
-                icon={ChangePasswordScreens[activeScreen].icon}
+                icon={isMt5 ? <MT5PasswordIcon /> : <DerivXPasswordIcon />}
```

Comment:
so much easier to read :)

Reply:
Yes but it was repeating so hence why I used `isMt5 ? <MT5PasswordIcon /> : <DerivXPasswordIcon />`